assets/src/styles/blocks/Counter.scss

```diff
@@ -30,59 +32,70 @@
   }
 
   .counter-target {
+    _-- {
+      color: inherit;
+      font-size: 2rem;
+      font-weight: bold;
+      margin-bottom: $space-lg;
+    }
     display: inline-block;
-
     line-height: 1;
-    font-size: 2rem;
-    font-weight: bold;
-    margin-bottom: $space-lg;
   }
 
   &.counter-style-1 {
-    .counter-target {
+    .counter-target _-- {
       font-size: 3rem;
       font-weight: bold;
       margin-bottom: $space-lg;
     }
   }
 
   .progress-container {
-    background-color: $grey-20;
-    height: 30px;
-    margin-bottom: $space-lg;
+    _-- {
+      display: block;
+      background-color: $grey-20;
+      height: 30px;
+      margin-bottom: $space-lg;
+    }
     overflow: hidden;
   }
 
   .progress-bar {
-    background-color: $dark-blue;
+    _-- {
+      background-color: $dark-blue;
+      transform: skewX(-30deg);
+    }
     height: 100%;
-    transform: skew(-30deg, 0deg);
     margin-left: -10px;
   }
 
-  .enform-progress-bar {
+  .enform-progress-bar _-- {
```

Comment:
Same here, you could use `background`

Reply:
Nice catch, forgot to change this one. Gradients could work well on the progress bar :ok_hand: